src/mpol/gridding.py

```diff
@@ -559,7 +559,7 @@ def get_dirty_image(
 
         return img.real, self.get_dirty_beam()
 
-    def to_pytorch_dataset(self):
+    def to_pytorch_dataset(self, check_visibility_scatter=True, max_scatter=np.sqrt(2)):
```

Comment:
sqrt(2) might be a little bit too large as a default argument (since some CASA routines are off by precisely this factor). What about 20%, e.g. 1.2?

Reply:
Oops, just saw this. Sure, I'll change that.